.github/workflows/updatecli.yaml

```diff
@@ -19,21 +19,19 @@ jobs:
         with:
           app_id: ${{ secrets.JENKINS_ADMIN_APP_ID }}
           private_key: ${{ secrets.JENKINS_ADMIN_APP_PRIVKEY }}
+      - name: Setup updatecli
+        uses: updatecli/updatecli-action@a149459e1feb1b410600dda2203cf1ee2afebf42 # v2
+
       - name: Diff
         continue-on-error: true
-        uses: updatecli/updatecli-action@a149459e1feb1b410600dda2203cf1ee2afebf42 # v2
-        with:
-          command: diff
-          flags: "--config ./updatecli/updatecli.d --values ./updatecli/values.github-action.yaml"
+        run: updatecli diff --config ./updatecli/updatecli.d --values ./updatecli/values.github-action.yaml
         env:
           # Github "generated" token is used because the repository secrets are not available in PRs.
-          UPDATECLI_GITHUB_TOKEN: ${{ steps.generate_token.outputs.token }}
+          UPDATECLI_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
@dduportal I've missed that but we should have either updated this comment too, or kept the generated token like in line 37 below isn't it?